pkg/inference/backend.go

```diff
@@ -30,8 +30,8 @@ func (m BackendMode) String() string {
 }
 
 type BackendConfiguration struct {
-	ContextSize int64    `json:"context_size,omitempty"`
-	RawFlags    []string `json:"flags,omitempty"`
+	ContextSize  int64    `json:"context-size,omitempty"`
+	RuntimeFlags []string `json:"runtime-flags,omitempty"`
```

Comment:
I don't know if this JSON nomenclature is used anywhere (I couldn't find it), but I've tried to make it consistent before it becomes used anywhere.